README.md

```diff
@@ -235,7 +236,7 @@ Option_group *app.add_option_group(name,description); // 🚧
 -app.add_mutable_set_ignore_case_underscore(... // 🆕 String only
 ```
 
-An option name must start with a alphabetic character, underscore, or a number. For long options, after the first character '.', and '-' are also valid. For the `add_flag*` functions '{' has special meaning. Names are given as a comma separated string, with the dash or dashes. An option or flag can have as many names as you want, and afterward, using `count`, you can use any of the names, with dashes as needed, to count the options. One of the names is allowed to be given without proceeding dash(es); if present the option is a positional option, and that name will be used on help line for its positional form. If you want the default value to print in the help description, pass in `true` for the final parameter for `add_option`.
+An option name must start with a alphabetic character, underscore, a number🚧, or one 🚧 of '$', '?', '#', '@', '%'. For long options, after the first character '.', and '-' are also valid characters. For the `add_flag*` functions '{' has special meaning. Names are given as a comma separated string, with the dash or dashes. An option or flag can have as many names as you want, and afterward, using `count`, you can use any of the names, with dashes as needed, to count the options. One of the names is allowed to be given without proceeding dash(es); if present the option is a positional option, and that name will be used on help line for its positional form. If you want the default value to print in the help description, pass in `true` for the final parameter for `add_option`.
```

Comment:
It might be okay to support $, it just seems a bit odd. I know that `-?` can be useful.

Reply:
/? is pretty standard for help in windows applications, and as I am converting other applications most of the time I just support help with -? and that failed the first time I tried it. So that is the most useful one.  The others I used -@ once as a shortcut for --address.  I guess I can see `$` being potentially problematic in bash.  If you wanted to remove all but '?'  that is probably fine.

Reply:
I'd like to drop `$` to avoid issues with bash shells (adding a `--$this` option would not behave as expected), similarly `#` and `%` (comments in bash, Variables in windows) `?` is safe, as is `@` I think. However, the code could support `$#%` still; I just don't think they should be advertised. The only issue is if you want to reserve these as shortcuts, like the `!{}` chars.